src/Nest/Cluster/TaskManagement/CancelTasks/CancelTasksResponse.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Runtime.Serialization;
 using Elastic.Transport;
+using Elastic.Transport.Products.Elasticsearch.Failures;
```

Comment:
Is this namespace a bit verbose? Could we loose "".Failures" in transport?

Reply:
Good callout opened: https://github.com/elastic/elastic-transport-net/issues/21 to track and discuss :)